Dockerfile

```diff
@@ -64,6 +64,9 @@ RUN set -a \
 
 RUN echo "IRB.conf[:USE_AUTOCOMPLETE] = false" > ./.irbrc
 
+# Install the OpenCV headless library
+RUN pip3 install opencv-python-headless --break-system-packages
```

Comment:
This is in response to a warning that Debian throws when this is deployed to Aptible. See https://stackoverflow.com/a/75696359 for more information.

Reply:
it sounds so hardcore!! smash the system!!